snuba/settings/__init__.py

```diff
@@ -82,7 +82,7 @@
 
 CLUSTERS: Sequence[Mapping[str, Any]] = [
     {
-        "host": os.environ.get("CLICKHOUSE_HOST", "127.0.0.1"),
+        "host": os.environ.get("CLICKHOUSE_HOST", "clickhouse.dev.local"),
```

Comment:
if this line is reverted, won't it be no longer necessary to add this name to `/etc/hosts`?

Reply:
As discussed on our call, I will replace `"clickhouse.dev.local"` with `"localhost"` and we _won't_ need edits to `/etc/hosts`. I will also update https://github.com/getsentry/sentry/pull/77125 to set the hostname to `localhost`. That should resolve your and @asottile-sentry's concerns.

Reply:
`localhost` is problematic as it can mean v6 `::0` on some machines and we don't run dualstack (in fact we run v4-only).  we previously removed all references to `localhost` specifically because it broke some sentry developer setups

Reply:
@asottile-sentry Hmm, how do you suggest we handle this then? The problem is that snuba admin is not run as a docker container when developing locally, it is spawned as a process by running `snuba admin` command. This is because snuba developers want to use the latest snuba on their Mac as opposed to the nightly build generated snuba (or whatever is in `master` branch). This means that in order to connect `snuba admin` (not in container) to clickhouse container locally, we need hostname of clickhouse that is recognized outside of docker. But it seems like neither `localhost` nor `clickhouse.dev.local` with additional path in `/etc/hosts` is a viable solution.
We don't have this problem in production because snuba admin is packaged in the `sentry_snuba` container.